content/en/docs/v3.5/op-guide/configuration.md

```diff
@@ -4,7 +4,7 @@ weight: 4050
 description: etcd configuration files, flags, and environment variables
 ---
 
-You can configure etcd through the following:
+You can configure etcd through one of the following:
```

Comment:
The issue with saying "one of the following" is that it implies only one option from the list can be used when users can mix and match configuration via environment variables and command line parameters.

Let's keep the current wording.